_haikus/do_mba.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: do_mba
+author: fcpimenta
+---
+
+This year I made my <br>
+MBA with DigitalOcean cloud <br>
+instead AWS and Azure <br>
```

Comment:
```suggestion
instead AWS <br>
```

Reply:
Hi Lisa, as always, As always, you shine and help us in any way you can. Thank you!!